content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+    - Dependendo do ambiente do cluster, isso pode expor o serviço apenas para a rede corporativa, ou pode expô-lo para a Internet. Pense se o serviço que está sendo exposto é seguro. Ele faz sua própria autenticação?
+    - Coloque Pods atrás de serviços. Para acessar um Pod específico de um conjunto de réplicas, como para depurar, coloque um rótulo exclusivo no Pod e crie um novo serviço que selecione este rótulo.
+    - Na maioria dos casos, não deve ser necessário para o desenvolvedor de aplicativos acessar diretamente nós através de seus IPs de nó.
+  - Acesse serviços, nós ou Pods usando o Verbo Proxy.
+    - Faz autenticação e autorização do servidor de API antes de acessar o serviço remoto. Use isto se os serviços não forem seguros o suficiente para expor à Internet, ou para obter acesso a portas no IP do nó, ou para depuração.
+    - Proxies podem causar problemas para algumas aplicações web.
+    - Só funciona para HTTP/HTTPS.
+    - Descrito [aqui](#manually-constructing-apiserver-proxy-urls).
+  - Acesse a partir de um nó ou Pod no cluster.
+    - Execute um Pod e, em seguida, conecte-se a um shell nele usando [kubectl exec](/docs/reference/generated/kubectl/kubectl-commands/#exec). Conecte-se a outros nós, Pods e serviços a partir desse shell.
+    - Alguns clusters podem permitir que você faça ssh para um nó no cluster. De lá, você pode ser capaz de acessar os serviços do cluster. Este é um método não padrão e funcionará em alguns clusters, mas não em outros. Navegadores e outras ferramentas podem ou não estar instalados. O DNS do cluster pode não funcionar.
+
+### Descobrindo serviços integrados
+
+Normalmente, existem vários serviços que são iniciados em um cluster pelo kube-system. Obtenha uma lista desses serviços com o comando `kubectl cluster-info`:
+
+```shell
+kubectl cluster-info
+```
+
+A saída é semelhante a esta:
+
+```
+Kubernetes master is running at https://192.0.2.1
+elasticsearch-logging is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy
+kibana-logging is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/kibana-logging/proxy
+kube-dns is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/kube-dns/proxy
+grafana is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy
+heapster is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-heapster/proxy
+```
+
+Isso mostra a URL proxy-verbo para acessar cada serviço. Por exemplo, este cluster tem o registro em nível de cluster habilitado (usando o Elasticsearch), que pode ser acessado em `https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/` se as credenciais adequadas forem passadas ou através de um proxy kubectl, por exemplo: `http://localhost:8080/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/`.
```

Comment:
```suggestion
Isso mostra a URL referente ao verbo proxy para acessar cada serviço. Por exemplo, este cluster tem _logs_ a nível de cluster habilitados (usando o Elasticsearch), que pode ser acessado em `https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/` se as credenciais adequadas forem passadas ou através do comando kubectl proxy, como por exemplo: `http://localhost:8080/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/`.
```

If I understood correctly, the intention here was to tell the user they can proxy the Kubernetes API using `kubectl proxy`, then access the service using the given URL.